### PR TITLE
[WIP] Use custom application name in tray menu (Kepka!)

### PR DESCRIPTION
**Pull request by minlexx**
TODO:
- other languages? use lang string with application name as parameter
- squash commits....

### CODE REVIEW

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.66-omhbptth completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.66-omhbptth) (commit https://github.com/procxx/kepka/commit/89cc6c69ec by @minlexx)

File: Telegram/SourceFiles/mainwindow.cpp [hunk 1]
```diff
@@ -166,14 +166,14 @@ void MainWindow::firstShow() {
 		: lng_enable_notifications_from_tray);
 
 	if (isLinux) {
-		trayIconMenu->addAction(lang(lng_open_from_tray), this, SLOT(showFromTray()))->setEnabled(true);
+		trayIconMenu->addAction(lang(lng_open_from_tray) + QCoreApplication::applicationName(), this, SLOT(showFromTray()))->setEnabled(true);
```
Comment (ForNeVeR) [inline Telegram/SourceFiles/mainwindow.cpp:169]: That's not a good idea. There're languages that use a different word order (e.g. in Azerbaijani that would be something like "Kepka/Telegram çıxın"). I suggest that we use the text templates here (e.g. "Quit {app_name}", like [here](https://github.com/minlexx/kepka/blob/34e3ac75e189da9bf84cd0588cd40afbf2185234/Telegram/Resources/langs/lang.strings#L99)) and use string format to prepare the resulting value.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

File: Telegram/SourceFiles/mainwindow.cpp (hunk 1 above)
Comment (minlexx) [reply Telegram/SourceFiles/mainwindow.cpp:169]: Yep, will fix later when I have time. Need to figure out how this translation system works... I see it automatically generates some .cpp and .h files

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.74-fljrmggr completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.74-fljrmggr) (commit https://github.com/procxx/kepka/commit/a16ae289f2 by @minlexx)

**Pull request closed without merging**